### DISCUSSION TITLE
Possible to hide object model and...

### DISCUSSION THREAD

**Discussion by Leanxp** [google/model-viewer#3397] [General]
Hello!
Is possible the Object model and only display the button "view in AR"?

![astronaut](https://user-images.githubusercontent.com/33395951/165628846-a7d17928-0389-47a1-a401-0815c0d08bb3.jpg)

**Comment by elalish** [comment]
Yes, but our UX research has found that most people don't accept the AR permissions if they haven't seen the 3D model first. You can just set `reveal="manual"` to keep the 3D model from loading.

**Reply by Leanxp** [reply]
The problem is that the object is still displayed in 3D even if I place it in this way.
![e](https://user-images.githubusercontent.com/33395951/165937400-8c05a677-8944-4746-9a04-f41928395369.jpg)

**Reply by elalish** [reply]
Can you post a link instead so I can repro?

**Reply by Leanxp** [reply]
OMG!
Sorry for not responding earlier!!! I've been very busy with work and university. And I did not receive the notification of your reply
You can find the index.html in: [https://github.com/Leanxp/Leanxp.github.io](url)

**Reply by elalish** [reply]
That page is a 404.

**Reply by Leanxp** [reply]
It is working now?
https://github.com/Leanxp/Leanxp.github.io

I can insert the code here:

```
<!doctype html>
<html lang="en">
  <head>
    <title>TFG</title>
    <meta charset="utf-8">
    <meta name="description" content="&lt;model-viewer&gt; template">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link type="text/css" href="./styles.css" rel="stylesheet"/>
    <!-- OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
    <script src="https://unpkg.com/focus-visible@5.0.2/dist/focus-visible.js" defer></script>
  </head>
  <body>
    <!-- <model-viewer> HTML element -->
    <model-viewer bounds="legacy" enable-pan src="Peluqueria.glb" ar-scale="auto" ar ar-placement="floor" ar-modes="webxr scene-viewer quick-look" camera-controls environment-image="neutral" poster="poster.webp" shadow-intensity="1">
      <div class="progress-bar hide" slot="progress-bar">
          <div class="update-bar"></div>
      </div>
      <button slot="ar-button" id="ar-button">
          Ver en AR (Augmented Reality)
      </button>
      <div id="ar-prompt">
          <img src="ar_hand_prompt.png">
      </div>
    </model-viewer>  
    <script src="script.js"></script>
    <!-- Loads <model-viewer> for browsers: -->
    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
  </body>
</html>
```

**Reply by elalish** [reply]
it's just a github repo. I was asking for a reproduction link; a page that your end users could see where I can reproduce the problem you mention. If you don't have one, you can create one by remixing our glitch: https://glitch.com/edit/#!/model-viewer

**Reply by Leanxp** [reply]
Ops, sorry 

Edit: The "reveal" problem has been solved!

**Reply by samaneh-kazemi** [reply]
Hi @Leanxp, thanks for sharing the glitch link. Can you please clarify the question here?

**Reply by Leanxp** [reply]
Sure, you only need to put "reveal"

https://imgur.com/34oKEwe

In the documentation it was explained: https://modelviewer.dev/docs/index.html#entrydocs-loading-attributes-reveal

**Reply by elalish** [reply]
Great, glad to hear! Perhaps you can mark one of these as the answer?
